activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/keycloak/KeycloakEnabledBaseTestIT.java

```diff
@@ -55,11 +55,11 @@ protected AccessTokenResponse authenticateUser() throws IOException {
 
     protected HttpHeaders getHeaders(String token) {
         HttpHeaders headers = new HttpHeaders();
-        headers.add("Authorization", "Bearer " + new String(token));
+        headers.add("Authorization", "Bearer " + token);
```

Comment:
@ryandawsonuk just curious... is this a normal practice... I would assume that spring will inject this for us.